commands/cli/parse_test.go

```diff
@@ -106,17 +106,25 @@ func TestOptionParsing(t *testing.T) {
 	test("-s foo", kvs{"s": "foo"}, words{})
 	test("-sfoo", kvs{"s": "foo"}, words{})
 	test("-s=foo", kvs{"s": "foo"}, words{})
-	test("-b", kvs{"b": ""}, words{})
-	test("-bs foo", kvs{"b": "", "s": "foo"}, words{})
+	test("-b", kvs{"b": true}, words{})
+	test("-bs foo", kvs{"b": true, "s": "foo"}, words{})
 	test("-sb", kvs{"s": "b"}, words{})
-	test("-b foo", kvs{"b": ""}, words{"foo"})
-	test("--bool foo", kvs{"bool": ""}, words{"foo"})
+	test("-b foo", kvs{"b": true}, words{"foo"})
```

Comment:
maybe also try:
```go
test("-b true", kvs{"b": true}, words{"true"})
test("-b false", kvs{"b": true}, words{"false"})
```
(i.e. we don't let bools be separated by space, as we cannot distinguish `-b <arg>` from `-b true`, right?)